engine/overworld/map_objects.asm

```diff
@@ -3061,3 +3061,132 @@ PRIORITY_HIGH EQU $30
 	dw wObject11Struct
 	dw wObject12Struct
 ; 5ae8
+
+CheckOccludingObjects:: ; here for space reasons
```

Comment:
This could go in its own engine/overworld/check_occluding_objects.asm file, and then get INCLUDEd in main.asm underneath this file's INCLUDE.